tests/main/mount-order-regression/task.yaml

```diff
@@ -0,0 +1,57 @@
+summary: Regression test for snap-update-ns mount order
+
+environment:
+    SNAP_NAME: test-content-layout-consumer
+
+details: |
+    A refactoring of snap-update-ns caused some snaps to fail to start.
+    We check that a typical GNOME snap using the content interface and layouts
+    has its filesystem laid out as expected.
+
+prepare: |
+    mkdir -p $SNAP_NAME/gnome-platform
```

Comment:
Please add ":
mkdir -p "$SNAP_NAME/gnome-platform"

Reply:
This apply for all the vars in the test. Shellcheck will be unhappy otherwise.

Reply:
Done!